bin/ch/ch.cpp

```diff
@@ -402,13 +405,23 @@ HRESULT CreateAndRunSerializedScript(const char* fileName, LPCSTR fileContents,
     IfJsErrorFailLog(ChakraRTInterface::JsGetCurrentContext(&current));
     IfJsErrorFailLog(ChakraRTInterface::JsSetCurrentContext(context));
 
+    // canonicalize that path name to lower case for the profile storage
+    // REVIEW: Assuming no utf8 characters here
```

Comment:
Oh I see this is already a review comment from below elsewhere in the code.  I suppose it hasn't been reported as a problem yet then and is okay to leave for now.